doc/source/collections/projecta.rst

```diff
@@ -45,7 +45,7 @@ The following lists key names mapped to its type and meaning for each entry.
 	:notes: list, any notes about the kickoff, optional
 	:status: string, status of the kickoff. Allowed values are {', '.join(PROJECTUM_STATI)}, optional
 
-		Allowed values:
```

Comment:
not sure the cause of these extra spaces provided throuhgout. I think this PR should only fix along `doc/source/api/...` right?